src/components/settings/playlist_settings/playlistsettingscomponent.module.css

```diff
@@ -37,3 +37,31 @@
 .toolbar {
 	max-height: 200px !important;
 }
+
+.toolbar_container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	gap: 1em;
+	margin: 15px 0;
+	animation: var(--fadeInAnimation);
+	position: sticky;
+	top: 0;
+	z-index: 100;
+	background-color: #fff;
+}
+
+.toolbar_container input {
+	width: 100%;
+	height: 16px;
+	background-color: transparent;
+	border-radius: 3px;
+	box-sizing: border-box;
+	border: 1px solid #4a4a4a;
+	font-size: 16px;
+	font-weight: 400;
+	color: #4a4a4a;
+	padding: 18px;
+	box-shadow: 0 0 15px rgba(0, 0, 0, 0.1);
+}
```

Comment:
Currently there is a blue outline when the input is being focussed. If possible, change this to `outline: none` for now as a general color for all outlines isn't implemented yet.